apps/tests/mbed_test/README.md

```diff
@@ -0,0 +1,14 @@
+Mbed example app
+================
+This is an example app that works with mbed os.
+
+It initializes with a signpost then blinks the debug LED.
+
+To use hardware that runs MBED OS you must:
+ 1) Copy this folder structure
+  - a .mbed file that describes the directories and targets
+  - a symblink to signpost-software/apps/support/mbed/mbed-os
```

Comment:
symblink -> symlink

Reply:
This really should be a fix to the mbed-port PR

Reply:
Okay I pushed a commit to that branch fixing the typo.